app/custom-sounds/client/admin/adminSounds.js

```diff
@@ -144,5 +144,20 @@ Template.adminSounds.events({
 		if ($audio && $audio[0] && $audio[0].play) {
 			$audio[0].play();
 		}
+	}, 'click .icon-pause-circled'(e) {
+		e.preventDefault();
+		e.stopPropagation();
+		const $audio = $(`audio#${ this._id }`);
```

Comment:
You must avoid jQuery dependency replacing ``$(`audio#{ this._id }`)`` with ``instance.find(`audio#{ this._id }`)`` (where `instance` is the second argument passed to this function).

Reply:
@tassoevan I think `instance.find()` is null here

Reply:
Ops, you're right, these `<audio>` tags are inserted somewhere else. Since they all have an ID, `document.getElementById(this._id)` will fit better.